docs/install/README.md

```diff
@@ -835,7 +846,7 @@ on Docker Hub for a list of available images.
 Install with kubectl:
 
 ```console
-kubectl apply -k github.com/k8ssandra/k8ssandra-operator/config/deployments/control-plane
+kubectl apply --server-side --force-conflicts -k github.com/k8ssandra/k8ssandra-operator/config/deployments/control-plane
```

Comment:
There is some overlap here with the changes in #437. We need to use `kustomize build | kubectl apply ...`. See #437 for background.